packages/orbit-components/src/Stack/index.tsx

```diff
@@ -26,7 +26,7 @@ import { JUSTIFY } from "../common/tailwind/justify";
 const shouldUseFlex = (props: CommonProps & Common.SpaceAfter) =>
   props.flex ||
   Object.keys(props)
-    .map(prop => ["spacing", "spaceAfter", "dataTest", "children"].includes(prop))
+    .map(prop => ["spacing", "spaceAfter", "dataTest", "children", "ariaLabel"].includes(prop))
```

Comment:
This puts to evidence how faulty this logic is 😞 any prop we add from now on we have to remind ourselves to come here and add it, otherwise the Stack won't use flex. Besides that, if one uses the media queries props (`desktop`, `tablet`, etc…) it will also consider it should not use flex… I wonder if that's expected 😬 

Do you think you can reconsider the logic here? I wouldn't ask you to take too much time, but give it a chance 🙏 If you feel it's too convoluted, let it be, as this fixes the immediate bug.

Reply:
As agreed, let's proceed with this as it is and address the bigger issue in different task after consulting with the team.